packages/fxa-settings/tailwind.config.js

```diff
@@ -0,0 +1,174 @@
+module.exports = {
+  purge: {
+    content: ['./src/**/*.tsx', './public/index.html'],
+    whitelist: [],
+  },
+  theme: {
+    screens: {
+      sm: '480px',
+      md: '768px',
+      lg: '1024px',
+      xl: '1441px',
+    },
+    fontSize: {
+      xs: '12px',
+      sm: '14px',
+      base: '16px',
```

Comment:
Should we call this `md` to match the other size conventions?

Reply:
Per our slack convo, gonna keep it for now as this is a [Tailwind convention](https://tailwindcss.com/docs/font-size/#app).